main.py

```diff
@@ -1,16 +1,24 @@
-# This is a sample Python script.
+import time
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
 
-# Press Shift+F10 to execute it or replace it with your code.
-# Press Double Shift to search everywhere for classes, files, tool windows, actions, and settings.
+# For detail instructions look here: 👀 https://sites.google.com/a/chromium.org/chromedriver/getting-started
+# and here https://selenium-python.readthedocs.io/getting-started.html
 
+# TODO:  before run of this project download right chrome driver
+#  👉 https://sites.google.com/a/chromium.org/chromedriver/downloads
+driver = webdriver.Chrome('./chromedriver')
 
-def print_hi(name):
-    # Use a breakpoint in the code line below to debug your script.
-    print(f'Hi, {name}')  # Press Ctrl+F8 to toggle the breakpoint.
-
-
-# Press the green button in the gutter to run the script.
-if __name__ == '__main__':
-    print_hi('PyCharm')
-
-# See PyCharm help at https://www.jetbrains.com/help/pycharm/
+driver.get("http://www.google.com")
+assert "Google" in driver.title
+elem = driver.find_element_by_name("q")
+elem.clear()
+elem.send_keys("d9nich")
+time.sleep(5)  # Let the user actually see something!
```

Comment:
try to avoid using time.sleep in your code, use Webdriver explicit wait instead

Reply:
The purpose of time.sleep was to give user ability to see something before script go further.

I also, don't fully understand **what should I use instead**.
 I have read [article about waits](https://selenium-python.readthedocs.io/waits.html), but they not cover my needs. Implicit waits, waits only some period of time before skipping, but it can be less. Explicit wait some period of time or specific event happened.

Reply:
For now, I have removed all time.sleep

Reply:
I think, this conversation is resolved